### PR TITLE
[#1478] Showing message.success after a profile(-related) save

### PR DESCRIPTION
**Pull request by alextreme**

### CODE REVIEW

File: src/open_inwoner/accounts/views/profile.py [hunk 1]
```diff
@@ -129,6 +129,7 @@ def get_object(self):
     def form_valid(self, form):
         form.save()
 
+        messages.success(request, _("Uw wijzigingen zijn opgeslagen"))
```
Comment (vaszig) [inline src/open_inwoner/accounts/views/profile.py:132]: `self.request`

File: src/open_inwoner/accounts/views/profile.py [hunk 2]
```diff
@@ -164,7 +165,7 @@ def get_object(self):
 
     def form_valid(self, form):
         form.save()
-
+        messages.success(request, _("Uw wijzigingen zijn opgeslagen"))
```
Comment (vaszig) [inline src/open_inwoner/accounts/views/profile.py:168]: `self.request`

File: src/open_inwoner/accounts/views/profile.py [hunk 3]
```diff
@@ -259,7 +260,7 @@ def get_form_kwargs(self):
...
```
Comment (vaszig) [inline src/open_inwoner/accounts/views/profile.py:263]: `self.request`

**Review by vaszig — CHANGES_REQUESTED** [review]
A small change and it's ready to go.

**Comment by alextreme** [comment]
More like: don't break stuff Alex :thisisfine:

**Pull request merged**